lib/widgets/screens/explore/explore.dart

```diff
@@ -1,57 +1,110 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:mm_flutter_app/__generated/schema/operations_user.graphql.dart';
+import 'package:mm_flutter_app/__generated/schema/schema.graphql.dart';
 import 'package:mm_flutter_app/constants/app_constants.dart';
+import 'package:mm_flutter_app/providers/user_provider.dart';
+import 'package:mm_flutter_app/providers/base/operation_result.dart';
+import 'package:mm_flutter_app/providers/models/explore_card_filters_model.dart';
+import 'package:mm_flutter_app/utilities/utility.dart';
 import 'package:mm_flutter_app/widgets/atoms/explore_filter.dart';
+import 'package:mm_flutter_app/widgets/atoms/profile_chip.dart';
 import 'package:mm_flutter_app/widgets/molecules/profile_quick_view_card.dart';
+import 'package:provider/provider.dart';
 
 import '../../../utilities/navigation_mixin.dart';
 
 class ExploreCardScroll extends StatefulWidget {
-  const ExploreCardScroll({super.key});
+  final UserProvider userProvider;
+  final ExploreCardFiltersModel exploreCardFilters;
+
+  const ExploreCardScroll({
+    super.key,
+    required this.userProvider,
+    required this.exploreCardFilters,
+  });
 
   @override
   State<ExploreCardScroll> createState() => _ExploreCardScrollState();
 }
 
 class _ExploreCardScrollState extends State<ExploreCardScroll> {
-  final int requestSize = 5;
-  List<bool> isSelected = [];
-  List<ProfileQuickViewInfo> cardInfo = [];
-
-  void _loadMoreRecommendations() {
-    // future todo:  load the actual backend request into cardInfo here instead
-    for (int i = 0; i < requestSize; i++) {
-      isSelected.add(false);
-      if (cardInfo.isEmpty) {
-        cardInfo.add(createRecommendedMentorExample());
-      } else if (cardInfo.length == 1) {
-        cardInfo.add(createRecommendedEntrepreneurExample());
-      } else {
-        cardInfo.add(createRegularMentorExample());
-      }
-    }
+  static const int profileIncrement = 5;
+
+  int _profiles = 0;
+  Future<OperationResult<List<RecommendedUser>>>? _recommendedUsers;
+
+  Future<void> _loadMoreRecommendations() async {
```

Comment:
You can make this `void _loadMoreRecommendations() { ... }` since you are not using `await` anywhere. Making this an async function breaks the setState pointed out in my other comment.

Reply:
Changing the signature as requested doesn't seem to solve the problem. I'm not sure that `FutureBuilder` can be triggered for assigning another value to the digested `Future`.